sdk/metricsadvisor/Azure.AI.MetricsAdvisor/perf/Azure.AI.MetricsAdvisor.Perf.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.AI.MetricsAdvisor.csproj" />
+    <ProjectReference Include="..\..\..\..\common\Perf\Azure.Test.Perf\Azure.Test.Perf.csproj" />
+  </ItemGroup>
```

Comment:
Make sure you can build this project when there are no binaries on disk.
I _think_ you also need a project reference to `Azure.Core.TestFramework.csproj`.